api/types/types.go

```diff
@@ -219,6 +219,7 @@ type Info struct {
 	ExperimentalBuild  bool
 	ServerVersion      string
 	ClusterStore       string
+	ClusterAdvertise   string
```

Comment:
the reason I added this is the fact that now we parse out the interface-name into ip-address 
and it provides the user ability to see which ip-address is being used.
Without that, they have no way of finding that out.
Also, it makes it consistent with ClusterStore config.

Please let me know if this should be removed.